syntax = "proto3";

package transaction;

option go_package = "./transaction";

service Transaction {
  rpc CreateUserBalance(CreateUserBalanceRequest) returns (CreateUserBalanceResponse) {}
}


// CreateUserBalance()

message CreateUserBalanceRequest {
  string user_id = 1;
}

message CreateUserBalanceResponse {
  string id = 1;
}
